src/dvc_data/index/index.py

```diff
@@ -540,6 +544,9 @@ def _load_from_storage(trie, entry, storage_info):
             return
         except (FileNotFoundError, ObjectFormatError):
             pass
+        except Exception:  # pylint: disable=W0703
+            # NOTE: this might be some random fs exception, e.g. auth error
+            logger.exception("failed to load %s", entry.key)
```

Comment:
will users see this? can it happen in a CLI case ...

overall looks a bit fragile

Reply:
For now just ignoring, later will likely add an onerror mechanism.